microservices/workflow/internal/consumer/rabbitmq.go

```
@@ -0,0 +1,481 @@
+				}
+				continue
+			}
+
+			// Process the message
+			go c.processMessage(ctx, msg)
+		}
+	}
+}
+
+// processMessage handles an individual message
+func (c *RabbitMQConsumer) processMessage(ctx context.Context, msg amqp.Delivery) {
```

golangci-lint (1.64.8)

cognitive complexity 67 of func `(*RabbitMQConsumer).processMessage` is high (> 20)

(gocognit)